docs/core/tools/elevated-access.md

```diff
@@ -0,0 +1,102 @@
+
+You can run commands as root as long as you don’t transition back and forth between root and a restricted account. For example, Docker containers run as root by default, so they have this characteristic.
+
+## Global tool installation
+
+The following instructions demonstrate the recommended way to install, run, and uninstall .NET Core tools that require elevated permissions to execute.
+
+# [Windows](#tab/windows)
+
+### Install the global tool
+
+If the folder `%ProgramFiles%\dotnet-tools` already exists, do the following to check whether the "Users" group has permission to write or modify that directory:
+
+* Right-click the `%ProgramFiles%\dotnet-tools` folder and select **Properties**. The **Common Properties** dialog box opens. 
+* Select the **Security** tab. Under **Group or user names**, check whether the “Users” group has permission to write or modify the directory. 
+* If the "Users" group can write or modify the directory, use a different directory name when installing the tools rather than *dotnet-tools*.
+
+To install tools, run the following command in elevated prompt. It will create the *dotnet-tools* folder during the installation.
+
+```cmd
+dotnet tool install PACKAGEID --tool-path "%ProgramFiles%\dotnet-tools".
+```
+
+### Run the global tool
+
+**Option 1** Use the full path with elevated prompt:
+
+```cmd
+"%ProgramFiles%\dotnet-tools\TOOLCOMMAND"
+```
+
+**Option 2** Add the newly created folder to `%Path%`. You only need to do this operation once.
+
+```cmd
+setx Path "%Path%;%ProgramFiles%\dotnet-tools\"
+```
+
+And run with:
+
+```cmd
+TOOLCOMMAND
+```
+
+### Uninstall the global tool
+
+In an elevated prompt, type the following command:
+
+```cmd
+dotnet tool uninstall PACKAGEID --tool-path "%ProgramFiles%\dotnet-tools"
+```
+
+# [Linux](#tab/linux)
+
+[!INCLUDE [elevated-access-unix](../../../includes/elevated-access-unix.md)]
+
+# [macOS](#tab/macos)
+
+[!INCLUDE [elevated-access-unix](../../../includes/elevated-access-unix.md)]
+
+---
+
+## Local tools
+
+Local tools are scoped per subdirectory tree, per user. When run elevated, local tools share a restricted user environment to the elevated environment. In Linux and macOS, this results in files being set with root user-only access. If the user switches back to a restricted account, the user can no longer access or write to the files. So installing tools that require elevation as local tools isn't recommended. Instead, use the `--tool-path` option and the previous guidelines for global tools.
+
+## Elevation during development
```

Comment:
one more question @wli3 - is this section specific to Linux and macOS?

Reply:
good question :p it should be able to apply to Windows in principle. I think the current state is good

Reply:
but isn't sudo something Unix specific?

Reply:
yes. so on windows, we should open cmd with admin instead. However, we haven't hit this situation for windows IoT yet. So i think we would add it when there is such case (and we can test it)